docs/support.md

```diff
@@ -0,0 +1,16 @@
+# Supported version policy
+
+This document describes which versions of Contour are supported by the Contour team.
+
+## Stable release
+
+Only the latest stable release is supported.
+The latest stable release is identified by the [Docker tag `:latest`](../tagging.md).
```

Comment:
We should also utilize Github's "Preview" release functionality, so `latest` docker tag matches up to github release's `latest` release.